### ISSUE TITLE
Распространения информации

### ISSUE THREAD

**Issue by GeorgeWeko** [free-belarus/info#30]
Можно попробовать охватить больше работников предприятий через рекламу. 
Посредством того же таргета можно попробовать добраться и сообщить о легальности забастовок, возможности получить юр. или мат. помощь. 
Можно попробовать рассмотреть данный вариант

**Comment by wasjawojasja** [comment]
@GeorgeWeko да, это нужно делать. Есть идеи как до них достучаться? Я так понимаю, что мы не можем пускать рекламу на БТ. Читают ли они Telegram? Мы можем распространять рекламу в Telegram, у нас в работе как минимум один рекламный ролик сейчас, и будут новые. Каналы распространения пока не понятны.

**Comment by wasjawojasja** [comment]
Я бы предложил информировать какие-то лекции типа: «[Почему восстали рабочие в Беларуси и что делать дальше?](https://youtu.be/xBY517c26hQ)» у Каца. Там он хорошо объясняет почему сокращение на заводах — на самом деле не проблема.

**Comment by wasjawojasja** [comment]
@GeorgeWeko у нас нашлись SMM-щики, попробую с ними обсудить эту идею. Попробую их пригласить сюда для обсуждения. Вернусь с новостями.

**Comment by GeorgeWeko** [comment]
@wasjawojasja, круто, сообщай по продвижению, можно тут накидывать варианты

**Comment by wasjawojasja** [comment]
Нашёл эту же идею, уже расписанную кем-то подробно: https://www.notion.so/Facebook-fb3c620f5585456b8e11d4ec2e9742af

Сам я подумал: ну ведь все, кто читает фейсбук — наверняка они все знают всю ситуацию, и достукиваться нам нужно до тех, кто не взаимодействует с интернетом активно.

**Comment by GeorgeWeko** [comment]
С фейсбуком понятно.
Но большая часть населения РБ/СНГ пока освоила ВК/Инст/ОК
Можно попробовать там, если пройдет модерацию

**Comment by f14a2cd** [comment]
> С фейсбуком понятно.
> Но большая часть населения РБ/СНГ пока освоила ВК/Инст/ОК
> Можно попробовать там, если пройдет модерацию

Пройдет, если там не будет агитации или призыва пойти на митинги.
